docker/Dockerfile

```diff
@@ -7,37 +7,33 @@
 # Pull base image.
 FROM anapsix/alpine-java
 
-# Install ElasticSearch.
-RUN apk update &&  apk add ca-certificates wget && apk add ca-certificates curl && update-ca-certificates
-RUN \
- mkdir /tmp/es && \
- cd /tmp/es && \
- wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-6.8.6.tar.gz && \
- tar xvzf elasticsearch-*.tar.gz && \
- rm -f elasticsearch-*.tar.gz && \
- mv /tmp/es/elasticsearch-* /elasticsearch && \
- rm -rf /tmp/es
+ENV ES_RELEASE 6.8.6
 
-# Define mountable directories.
-VOLUME ["/data"]
+# Install ElasticSearch.
+RUN apk update && apk add --no-cache ca-certificates \
+    ca-certificates curl && \
```

Comment:
the package `ca-certificates` is repeated, can u please fix this?

Reply:
Good spot @sanacl - done.